auth/native/data.go

```diff
@@ -15,8 +30,17 @@ need to use the following token to finish this process:
 {{.Token}}
 
 If you think this is email is wrong, just ignore it.`))
-
-var passwordResetConfirm = template.Must(template.New("reset").Parse(`Subject: [tsuru] Password successfully reset
+}
+
+func getEmailResetPasswordSucessfullyTemplate() (*template.Template) {
+	templateFile, _ := config.GetString("email-reset-password-successfully-template-file")
```

Comment:
This `config` parameter is to long. I believe that `reset-password` and `reset-password-sucessfully` is a better option. Another good option is to use namepscaes